### DISCUSSION TITLE
怎样禁止pkg-config找到的包？

### DISCUSSION THREAD

**Discussion by wsw0108** [xmake-io/xmake#1502] [Q&A] [answered]
需要这个特性来测试。(`{configs = {system = false}}`不行。

或者有没有什么办法强制使用xmake的包(`xmake::pkgname`)不行。

**Comment by waruqi** [comment, marked as answer]
仔细看文档，是 {system = false} , 不在  configs 里面

**Reply by wsw0108** [reply]
好吧，和shared搞混了。

**Comment by wsw0108** [comment]
弄错了。
